src/templates/antora/util.ts

```diff
@@ -0,0 +1,262 @@
+    }
+    return descriptions;
+  }, {});
+};
+
+const getNavFile = (types: Type[]): NavFile => {
+  const namespaces = getNamespacesFromTypes(types);
+  const pages = Object.entries(namespaces).map(([ url, title ]): NavFile => {
+    const innerPages = types.filter((type) => {
+      const fullName = type.fullName.toLowerCase();
+      return getNamespaceFromFullName(fullName) === url;
+    }).map((type): NavFile => {
+      return { title: type.fullName, path: type.fullName.toLowerCase() };
+    });
+
+    return {
+      title,
+      path: url,
+      pages: innerPages
+    };
+  });
+
+  return {
+    title: 'API Reference',
+    path: BASE_PATH,
+    pages
+  };
+};
+
+const navToAdoc = (indexPage: NavFile, structure: ExportStructure): string => {
+  // Api index nav page top level
+  let adoc = navLine(indexPage.title, 1);
+  if (indexPage.pages) {
+    indexPage.pages.forEach((namespace) => {
+      adoc += navLine(namespaceNavLine(namespace, structure), 2);
+      if (namespace.pages) {
+        namespace.pages.forEach((pageFile) => {
+          adoc += navLine(pageFileNavLine(pageFile, structure), 3);
+        });
+      }
+    });
+  }
+  return adoc;
+};
+
+const generateNavPages = (indexPage: NavFile, structure: ExportStructure): PageOutput[] => {
+  const navPages = [] as PageOutput[];
+  navPages.push({
+    type: 'adoc',
+    filename: '_data/nav.yml',
+    content: YAML.dump(indexPage)
+  });
+
+  const adocNav = navToAdoc(indexPage, structure);
+  navPages.push({
+    type: 'adoc',
+    filename: '_data/moxiedoc_nav.adoc',
+    content: adocNav
+  });
+  return navPages;
+};
+
+const legacyIndexToAdoc =
+(namespace: NavFile, template: HandlebarsTemplateDelegate, descriptions: Record<string, string>, structure: ExportStructure): string => {
+  const keywords = [ getApiFromFullName(namespace.title) ];
+  const indexPageLines = [
+    '\n== ' + namespaceDescriptions[namespace.title.toLowerCase()] + '\n\n'
+  ];
+  if (namespace.pages) {
+    indexPageLines.push(
+      '[cols="1,1"]\n',
+      '|===\n\n'
+    );
+    namespace.pages.forEach((pageFile) => {
+      keywords.push(getNameFromFullName(pageFile.title));
+      indexPageLines.push('a|\n');
+      indexPageLines.push('[.lead]\n');
+      indexPageLines.push(pageFileLegacyIndexLine(pageFile, structure) + '\n\n');
+      const description = descriptions[pageFile.path];
+      indexPageLines.push(description + '\n\n');
+    });
+    if (namespace.pages.length % 2 !== 0) {
+      indexPageLines.push('a|\n');
+    }
+    indexPageLines.push('|===\n');
+  }
+  const data = {
+    fullName: namespace.title,
+    desc: namespaceDescriptions[namespace.title.toLowerCase()],
+    keywords: keywords.join(', ')
+  };
+  const adoc = template(data) + indexPageLines.join('');
+  return adoc;
+};
+
+const generateLegacyIndexPages =
+(indexPage: NavFile, sortedTypes: Type[], memberTemplate: HandlebarsTemplateDelegate, structure: ExportStructure): PageOutput[] => {
```

Comment:
Likewise here for the code style
```suggestion
const generateLegacyIndexPages = (
  indexPage: NavFile,
  sortedTypes: Type[],
  memberTemplate: HandlebarsTemplateDelegate,
  structure: ExportStructure
): PageOutput[] => {
```